client/asset/btc/btc.go

```diff
@@ -1594,7 +1598,13 @@ func (btc *ExchangeWallet) spendableUTXOs(confs uint32) ([]*compositeUTXO, map[o
 	utxos := make([]*compositeUTXO, 0, len(unspents))
 	utxoMap := make(map[outPoint]*compositeUTXO, len(unspents))
 	for _, txout := range unspents {
-		if txout.Confirmations >= confs && txout.Safe {
+		if txout.Confirmations >= confs && txout.Safe && txout.Spendable {
+			// TODO when the map keys change to the outpoint struct:
+			// opID := outpointID(txout.TxID, txout.Vout)
+			// if btc.fundingCoins[opID] != nil {
+			// 	btc.log.Warnf("Skipping known order-funding coin %v returned by listunspent!", opID)
+			// 	continue
+			// }
```

Comment:
Resolve now?

Reply:
Done, although a similar check in dcr is slightly more nuanced.  I have a stash with some other related changes that might make it to PR soon.